static/css/components/work.less

```diff
@@ -1,13 +1,26 @@
 @import (less) "base/dl.less";
 
 .workDetails {
-  padding-bottom: 15px;
+  padding: 20px 0;
+  .section {
+    text-align: justify;
+  }
 }
 
 // Left (editionCover)
-div.editionCover {
+.editionCover {
   min-height: 10px;
   padding-right: 1.5em;
+
+  .illustration {
+    transform: perspective(800px) rotateX(10deg);
+    transition: 0.8s;
+  }
+  .illustration:hover {
+    transform: perspective(465px) rotateX(0deg) rotateY(-10deg);
+    filter: alpha(opacity=80);
```

Comment:
This line doesn't look like valid LESS; It can be deleted.

Reply:
Which line? `filter: alpha(opacity=80);`? Should it be replaced with `opacity: .8;`?

Reply:
fixed